src/executors/mod.rs

```diff
@@ -69,14 +66,25 @@ pub async fn execute_middleware_function<'a>(
                 request.insert("queries", queries_clone.into_py(py));
                 // is this a bottleneck again?
                 request.insert("headers", headers.clone().into_py(py));
-                // request.insert("body", data.into_py(py));
+                request.insert("body", data.into_py(py));
+                response_dict.insert("headers", response_headers.into_py(py));
+                response_dict.insert("status", response_status_code.into_py(py));
+                response_dict.insert("body", response_body.into_py(py));
+                info!("{:?}", request);
+                info!("{:?}", response_dict);
+                info!("{:?}", number_of_params);
```

Comment:
Do we need these logs?

Reply:
no we don't these were just for debugging purposes